tests/test_servefile.py

```diff
@@ -69,7 +90,8 @@ def _datadir(data, path=None):
     return _datadir
 
 
-def make_request(path='/', host='localhost', port=8080, method='get', protocol='http', **kwargs):
+def make_request(path='/', host='localhost', port=SERVEFILE_DEFAULT_PORT, method='get', protocol='http', **kwargs):
+    kwargs.setdefault("timeout", 5)
```

Comment:
Any reason why `def make_request([..], timeout=5, **kwargs)` would not work here?
`

Reply:
That'll probably make it clearer.